frontend/src/components/table/BookingsTable.tsx

```diff
@@ -91,6 +95,7 @@ export default function BookingsTable({
                     link.href = url;
                     link.click();
                     link.href = '';
+                    addAlert('Sucessfully Exporting the Excel!');
```

Comment:
"Successfully exported the Excel file"

This one has a typo in successfully, and isn't grammatically correct (s/exporting/exported).

Reply:
Thank you very much for improving my Eng. 👍 
I will make a change to `Successfully exported the ADM322 Excel file`